db/migrate/20220407210610_add_placement_group_id_to_vm.rb

```diff
@@ -0,0 +1,18 @@
+class AddPlacementGroupIdToVm < ActiveRecord::Migration[6.0]
```

Comment:
The migration name here is confusing, sounds like we are just adding an ID to a table but we are actually creating a new table.

Recommend 2 migrations, 1. `CreatePlacementGroups` where you do https://github.com/ManageIQ/manageiq-schema/pull/644/files#diff-878bd2c16dde24ca462a9e27431187f56aacd0df732e68f1e127240b4f2c51bcR5-R16 then 2. where you do https://github.com/ManageIQ/manageiq-schema/pull/644/files#diff-878bd2c16dde24ca462a9e27431187f56aacd0df732e68f1e127240b4f2c51bcR3